tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -71,11 +73,13 @@ internal static void CloseAndFlush()
         internal static void Reset()
         {
             LoggingLevelSwitch.MinimumLevel = DefaultLogLevel;
+            LibDatadog.Logger.Instance.SetLogLevel(DefaultLogLevel);
```

Comment:
I think we should _probably_ move these calls to the _callsites_ for these methods, e.g. in `GlobalSettings.SetDebugEnabledInternal()` and `GlobalSettings.Reset()` 🤔  We may run into testing issues otherwise